pygfx/renderers/wgpu/shaders/textshader.py

```diff
@@ -42,7 +42,11 @@ def get_bindings(self, wobject, shared):
         ]
 
         tex = shared.glyph_atlas_texture
-        sampler = GfxSampler("linear", "clamp")
+        # We use a repeat since we have a half pixel padding on each glyph
+        # however, we only have half on the '0' edge of the x and y dimensions
+        # therefore on the high value edge, we need to 'wrap' back to the beginning
+        # to get the '0' from the 0 edge.
+        sampler = GfxSampler("linear", "repeat")
```

Comment:
I don’t get it. This should have worked…..

do we provide sampling options in the two dimensions????

Reply:
This seems to be working now.